src/libcore/iter/mod.rs

```diff
@@ -1094,6 +1107,12 @@ impl<A, B> ZipImpl<A, B> for Zip<A, B>
         })
     }
 
+    #[inline]
+    default fn nth(&mut self, n: usize) -> Option<Self::Item>
+    {
```

Comment:
Minor nit: why this `{` is on a new line 😄

Reply:
Now that is an excellent question  :sweat_smile: